### PR TITLE
add blog proposal for video

### PR DESCRIPTION
**Pull request by egekorkan**
Here is the first version of the blog post proposal.

I am not sure about the last part, i.e. acknowledgment. It should be decided together I think. Also, there should be the video link/embed put in there but I am not sure how it is done in the W3C blog.

After putting in the blog, we should also tweet about it.

### CODE REVIEW

File: proposals/blogProposal.md
Comment (danielpeintner) [inline proposals/blogProposal.md:1]: "locked-in to framework" --> "locked intoframeworks" ?
"smart home or more!" --> "smart home and more!" ?

File: proposals/blogProposal.md
Comment (danielpeintner) [inline proposals/blogProposal.md:5]: "to come up with an amazing video!you" --> "to come up with our amazing video. [See your self!](TODO-Link)"

Comment (egekorkan) [reply proposals/blogProposal.md]: Done

File: proposals/blogProposal.md
Comment (egekorkan) [reply proposals/blogProposal.md]: Done mostly. Just wasn't sure about the word visible since nothing was not visible before. I wanted to use something like "can be better understood" but that was also not super positive :(

File: proposals/blogProposal.md
Comment (relu91) [inline proposals/blogProposal.md:1]: I am not sure with visible either. Alternatives that come to my mind: `described`, `available` or simply saying `are now on a new animation...`

**Review by relu91 — CHANGES_REQUESTED** [review]

File: proposals/blogProposal.md
Comment (koalie) [reply proposals/blogProposal.md:1]: How about "illustrated"?

Comment (koalie) [reply proposals/blogProposal.md]: "Explained" would be another possible alternative to "visible"

**Review by koalie — COMMENTED** [review]
I have done an editorial pass and suggested two words to replace "visible".

I have one remaining question: what does "-- To be decided" mean?

**Review by relu91 — APPROVED** [review]
Right, much better now. My preference would be explained over illustrated but I leave the choice to native speakers!

**Comment by danielpeintner** [comment]
>  have one remaining question: what does "-- To be decided" mean?

I believe this was a **TODO** for us whether we want to have a statement like "who supported the video to happen .."

During the marketing call we made the decision that we do *NOT* want to have any such statement. It should be neutral.

**Comment by koalie** [comment]
Hi all, if anyone needs help to move the blog post draft to the W3C blog post, let me know.
W3C Groups chairs are among those who can blog directly; see the [manual](https://www.w3.org/2007/12/qa-blog-help/index.xhtml) for everything about it, but ping me if it's easier.

File: proposals/blogProposal.md
Comment (sebastiankb) [inline proposals/blogProposal.md:1]: @mmccool would go for illustrated

Comment (sebastiankb) [inline proposals/blogProposal.md:1]: "see how any web developer " --> "see how a web developer"

File: proposals/blogProposal.md
Comment (sebastiankb) [inline proposals/blogProposal.md:5]: we should remove the names of the companies.

Comment (sebastiankb) [reply proposals/blogProposal.md:5]: "Initiative team for their feedback, as well as Siemens AG and Intel Inc. for the ..." --> "Initiative team for their feedback and the ..."

**Review by sebastiankb — CHANGES_REQUESTED** [review]

**Comment by sebastiankb** [comment]
From today's marketing call:
- we will ask @egekorkan to fix the issues above
- @sebastiankb will publish the blog post at W3C side

**Comment by egekorkan** [comment]
I have incorporated the feedback. I have removed the mention of WebCastor since I couldn't find their mention anywhere.

**Review by sebastiankb — APPROVED** [review]

**Review by danielpeintner — APPROVED** [review]

**Comment by egekorkan** [comment]
Should we merge this after approvals or wait for Marketing call where we also agree on showing it in the Main Call?

**Comment by sebastiankb** [comment]
from my side +1 for bringing the post to the public. @mmccool what do you think?

File: proposals/blogProposal.md
Comment (mmccool) [inline proposals/blogProposal.md:1]: Delete "that we have been working on for the past few months."   Not relevant, and it makes us look slow.  "You can see how" -> "The animation shows how", "allows the experience of the developers" -> "allows developer experience and skills"

**Comment by mmccool** [comment]
In the main call on Apr 7 we reviewed and decided to merge (into the repo) with a few edits by myself.  Also, as chair, and deciding this under the "tweet" review policy (also appropriate to blogs, I think), I am authorizing publication (after my changes).  Please review (quickly) if you have any further changes, however I think we should proceed with publication now.

**Pull request merged**
